index.html

```diff
@@ -2,18 +2,29 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Document</title>
+    <title>quiz app</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
+
   <body>
     <header>
-      <h2>Quiz App</h2>
+      <h1>Quiz App</h1>
     </header>
-    <section>
-      <h3>Very hard question</h3>
-      <h4>show me the answer</h4>
-      <h6 class="tag1">bookmark1</h6>
-      <h6 class="tag2">bookmark2</h6>
-    </section>
+
+    <main>
+      <section class="q1">
+        <h2>Question 1</h2>
+        <button type="button" class="button-style">Show Answer</button>
+        <span class="t1">tag1</span>
+        <span class="t2">tag2</span>
```

Comment:
I think also your tags don't need different classes. Or no class, since you don't use the classes for styling yet, or?